### ISSUE TITLE
Question: Descendants function

### ISSUE THREAD

**Issue by marekliska1** [matthiask/django-tree-queries#43]
Hello, did I understand correctly that the "descendants function" only returns descendants in the first line? Is there no way to get all descendants of descendants? In other words the whole tree structure?

**Comment by matthiask** [comment]
I don't think that's accurate. There may be bugs, but `descendants` really shouldn't do the same thing as `.children.all()` does.

Here's a test, too: https://github.com/matthiask/django-tree-queries/blob/ebb257e8b73f9bf267d5e8eee99f53b23e6a4f4a/tests/testapp/test_queries.py#L250-L262

**Comment by marekliska1** [comment]
Yes, you're right. I'm sorry, it was a mistake on my part. It's working as it should, which means I get all the descendants up to the end of the tree structure. Thanks for this library!

**Comment by matthiask** [comment]
Thanks, you're welcome!
